packages/server/integrations/JiraServerTaskIntegrationManager.ts

```diff
@@ -17,7 +17,7 @@ export default class JiraServerTaskIntegrationManager implements TaskIntegration
     this.provider = provider
   }
 
-  private getManager() {
+  public getAPI() {
```

Comment:
Why change to `getAPI` out of interest? It looks like we're using `manager` in other parts of the codebase

Reply:
Renamed to `getApiManager` for now